lib/sheets-data-builder.rb

```diff
@@ -29,7 +29,7 @@ def url
 
   def grouped_data(refresh = false)
     @grouped_data   = nil if refresh
-    @grouped_data ||= raw_data(refresh).each_slice(GROUP_SIZE)
+    @grouped_data ||= raw_data(refresh).each_slice(GROUP_SIZE - 1)
```

Comment:
Yep - it works. It works here because each subsequent piece works with the size minus one logic. The headers are parsed in a separate method which aren't using the minus one.